.circleci/config.yml

```diff
@@ -28,41 +28,23 @@ commands:
             docker login -u "${DOCKERHUB_USER}" -p "${DOCKERHUB_PASS}"
             docker push "${DOCKERHUB_REPO}":<< parameters.image_tag >>
 jobs:
-  build-py2:
+  build:
     machine: true
     working_directory: ~/addons-server
     steps:
       - checkout
       - build_and_push_container_image:
           image_tag: latest
-          dockerfile: "Dockerfile.python2.deploy"
+          dockerfile: "Dockerfile.deploy"
```

Comment:
This reverts us back to having a single `Dockerfile.deploy` that builds an image called `latest` (and below `"${CIRCLE_TAG}"` for tags). So this will need ops to change the jenkins pipeline to match.